examples/4-tests/example_test.go

```diff
@@ -353,10 +353,18 @@ func Test_7InputOutputWithHeaders(t *testing.T) {
 	proc, _ := goka.NewProcessor([]string{}, goka.DefineGroup("group",
 		goka.Input("input", new(codec.String), func(ctx goka.Context, msg interface{}) {
 			processorHeaders = ctx.Headers()
-			ctx.Emit("output", ctx.Key(), fmt.Sprintf("forwarded: %v", msg), goka.WithCtxEmitHeaders(
-				map[string][]byte{
-					"Header1": []byte("to output"),
-				}))
+			ctx.Emit("output", ctx.Key(), fmt.Sprintf("forwarded: %v", msg),
+				goka.WithCtxEmitHeaders(
+					map[string][]byte{
+						"Header1": []byte("to output"),
+						"Header2": []byte("to output2"),
+					}),
+				goka.WithCtxEmitHeaders(
+					map[string][]byte{
+						"Header2": []byte("to output3"),
+						"Header3": []byte("to output4"),
```

Comment:
Is it header `Header2, Header3`, or `Header3 Header4`?

Reply:
ok I admit the values are a bit misleading, but the idea here was to test the overwriting of header values. `Header2` is in both header-options and gets overwritten by the "last version".